opencart 1.5.6.4/upload/admin/view/template/module/smaily_for_opencart.tpl

```diff
@@ -78,11 +78,11 @@
                 <td>
                   <?php echo $username_title; ?>
                 </td>
-                <td> 
-                  <input 
+                <td>
+                  <input
                     type="text"
                     name="smaily_for_opencart_username"
-                    placeholder="<?php echo $username_placeholder; ?>"
+                    placeholder="<?php echo $useroptin_form_name_placeholder; ?>"
```

Comment:
You missed this variable name when reverting back the changes. It should be `$username_placeholder`.

Reply:
Ok. I have no idea why I replaced it at all in the first place.